README.md

```diff
@@ -24,6 +24,7 @@ Part of the Foreman installer: <https://github.com/theforeman/foreman-installer>
 | 2.x            | 1.5 - 1.10     |                                                     |
 | 1.x            | 1.4 and older  |                                                     |
 
+21.x `foreman_proxy::remote_execution::ssh` now requires [smart_proxy_remote_execution_ssh](https://github.com/theforeman/smart_proxy_remote_execution_ssh) >= 0.5.0, which replaced the old `async_ssh` option with a more general configuration of the operation `mode`
```

Comment:
Thanks for this.

Looking at the current release we'd still be targeting 3.1 and newer in version 21.x.

It looks like Foreman 3.1 has version REX SSH 0.5.1 for RPMs but 0.4.1 for deb. If we update the deb packages in 3.1 it would work on all supported versions.

@adamruzicka is this intentional?

Reply:
deb/3.1 should have at least 0.5.1 per[1]. And at least buster has it in the repos

[1] - https://github.com/theforeman/foreman-packaging/pull/7445

Reply:
Hmm, then I wonder why my search yesterday didn't show this. Perhaps it was too late and I missed something. In that case I think this is a good change. Thanks!